### PR TITLE
GitLab now supports 2FA (software: OTP)

### PR DESCRIPTION
**Pull request by rugk**
GitLab [let me know](https://twitter.com/gitlab/status/604405976884731905) that they are now supporting 2FA, so here I added it.
For now I've added their blog post as a documentation.

### CODE REVIEW

**Comment by jamcat22** [comment]
Is there any other documentation? Or just the blog post

**Comment by rugk** [comment]
Well I've already asked this... :smiley: 
https://twitter.com/rugkme/status/604409881483952129

**Comment by mxxcon** [comment]
If there's doc link, update it, otherwise :+1:

File: _data/developer.yml [hunk 1]
```diff
@@ -101,7 +101,9 @@ websites:
       url: https://gitlab.com
       twitter: gitlab
```
Comment (Carlgo11) [inline _data/developer.yml]: remove the twitter handle please

Comment (rugk) [inline _data/developer.yml]: Done.

**Comment by rugk** [comment]
@mxxcon
[Gitlab said](https://twitter.com/gitlab/status/604430601417031680) there is no doc link, so I think the blog is the only thing we have for now.

**Pull request merged**